### DISCUSSION TITLE
Reporting bug - Skeletal Horror Fear spell

### DISCUSSION THREAD

**Discussion by hnkowned4** [chromiecraft/chromiecraft#4085] [General]
In Duskwood, next to Darkshire there are these Skeletal Horror mobs who uses a Terror or Fear ability... Well, they spam it, they fear you into other mobs and then its just nonstop fears until you end up dead. Pretty sad not enjoyable.

Please fix this issue if possible. Thank you for taking the time reading this report.

Kind regards,
Loathe (ingame name)

**Comment by 55Honey** [comment]
Hello @hnkowned4 ,
thanks for your report.

Please make use of https://github.com/chromiecraft/chromiecraft/issues to report a bug, thanks!
